### PR TITLE
Add brightness to the LED mode.

### PR DESCRIPTION
**Pull request by benallard**
So, here my progress as a PoC.

It doesn't seems to be working properly as the LED is silent when not at full brightness, and I haven't touched at the devices I couldn't test.

### CODE REVIEW

File: src/led/led.h [hunk 1]
```diff
@@ -31,28 +31,37 @@ enum {
...
-    BLINK_MOUNTED     = (LED_COLOR_GREEN << LED_COLOR_SHIFT) | (250 << LED_ON_SHIFT) | (250 << LED_OFF_SHIFT),
-    BLINK_SUSPENDED   = (LED_COLOR_BLUE << LED_COLOR_SHIFT) | (500 << LED_ON_SHIFT) | (1000 << LED_OFF_SHIFT),
-    BLINK_PROCESSING  = (LED_COLOR_GREEN << LED_COLOR_SHIFT) | (50 << LED_ON_SHIFT) | (50 << LED_OFF_SHIFT),
-    BLINK_BUTTON      = (LED_COLOR_YELLOW << LED_COLOR_SHIFT) | (1000 << LED_ON_SHIFT) | (100 << LED_OFF_SHIFT),
-
-    BLINK_ALWAYS_ON   = UINT32_MAX,
-    BLINK_ALWAYS_OFF  = 0
+    MODE_NOT_MOUNTED = (HALF_BTNESS << LED_BTNESS_BITS) | (LED_COLOR_CYAN << LED_COLOR_SHIFT) | LED_ON_NO_BLINK,
+    MODE_MOUNTED     = (HALF_BTNESS << LED_BTNESS_BITS) | (LED_COLOR_GREEN << LED_COLOR_SHIFT) | LED_ON_NO_BLINK,
+    MODE_SUSPENDED   = (HALF_BTNESS << LED_BTNESS_BITS) | (LED_COLOR_BLUE << LED_COLOR_SHIFT) | (1000 << LED_ON_SHIFT) | (2000 << LED_OFF_SHIFT),
```
Comment (benallard) [inline src/led/led.h]: The Shift is wrong there. Should be LED_BTNESS_SHIFT instead of LED_BTNESS_BITS

**Comment by polhenarejos** [comment]
I see two problems with this PR:
1. It reduces the brightness to the half, which might be undesirable to other users.
2. It creates the same pattern (no blink) when unmounted and mounted if the board is single led. Blink is necessary for these boards.

Maybe you want to correct this.

**Comment by polhenarejos** [comment]
I tested and it is always off.

**Comment by benallard** [comment]
There was issues with integer divisions. I just pushed an improved version.

**Pull request merged**